### PR TITLE
fix: heartbeat status is STOP_STATUS after ha disable

### PR DESCRIPTION
**Pull request by LUAgam**
Reason:  
  BUG inner-1847
Type:  
  BUG
Influences：  
fix: heartbeat status is STOP_STATUS after ha disable

### CODE REVIEW

File: src/main/java/com/actiontech/dble/backend/datasource/PhysicalDbGroup.java [hunk 1]
```diff
@@ -458,7 +458,7 @@ private List<PhysicalDbInstance> getRWDbInstances(boolean includeWrite) {
             if (ds.isAlive() && (!checkSlaveSynStatus() || ds.canSelectAsReadNode())) {
                 okSources.add(ds);
             } else {
-                if (!ds.isAlive()) {
```
Comment (yanhuqing666) [inline src/main/java/com/actiontech/dble/backend/datasource/PhysicalDbGroup.java:461]: I think it also need to print log when `!ds.isAlive()`

Comment (LUAgam) [reply src/main/java/com/actiontech/dble/backend/datasource/PhysicalDbGroup.java:461]: too many logs printed when !ds.isAlive()
inner-1845

**Comment by LUAgam** [comment]
need update: reduce the frequency of log printing

**Review by PanternBao — APPROVED** [review]
lgtm

**Pull request merged**